packages/form-js-carbon-styles/src/carbon-styles.js

```diff
@@ -0,0 +1,993 @@
+        ${getSelectArrowStyles({
+    arrowRightPosition: 'var(--cds-spacing-05)',
+    color: theme.iconDisabled,
+  })}
+      }
+
+      .fjs-has-errors.fjs-form-field-select .fjs-input-group:focus-within {
+        outline: 2px solid var(--cds-focus);
+        outline-offset: -2px;
+      }
+
+      .fjs-has-errors.fjs-form-field-select .fjs-input-group {
+        outline: 2px solid var(--cds-text-error);
+        outline-offset: -2px;
+      }
+    }
+  `}
+`;
+
+const REMAINING_INPUTS = css`
+  .fjs-container {
+    .fjs-form-field-textfield .fjs-input-group,
+    .fjs-form-field-datetime .fjs-input-group,
+    .fjs-textarea,
+    .fjs-taglist,
+    .fjs-form-field-select.fjs-disabled .fjs-input-group {
+      ${getBaseInputStyles({ height: '2.5rem' })}
+    }
+
+    .fjs-form-field-textfield .fjs-input-group,
+    .fjs-form-field-datetime .fjs-input-group {
+      &:focus-within {
+        outline: 2px solid var(--cds-focus);
+        outline-offset: -2px;
+      }
+    }
+    .fjs-form-field-textfield .fjs-input,
+    .fjs-form-field-datetime .fjs-input {
+      background-color: var(--cds-field, var(--cds-field-01));
+      color: var(--cds-text-primary);
+    }
+
+    .fjs-has-errors.fjs-form-field-number .fjs-input-group:focus-within,
+    .fjs-has-errors.fjs-form-field-textarea .fjs-textarea:focus,
+    .fjs-form-field-textfield.fjs-has-errors .fjs-input-group:focus-within,
+    .fjs-form-field-textfield.fjs-has-errors .fjs-input-group:focus,
+    .fjs-form-field-datetime.fjs-has-errors .fjs-input-group:focus-within,
+    .fjs-form-field-datetime.fjs-has-errors .fjs-input-group:focus {
+      outline: 2px solid var(--cds-focus);
+      outline-offset: -2px;
+    }
+
+    .fjs-has-errors.fjs-form-field-number .fjs-input-group,
+    .fjs-has-errors.fjs-form-field-textarea .fjs-textarea,
+    .fjs-form-field-textfield.fjs-has-errors .fjs-input-group,
+    .fjs-form-field-textfield.fjs-has-errors .fjs-input-group,
+    .fjs-form-field-datetime.fjs-has-errors .fjs-input-group,
+    .fjs-form-field-datetime.fjs-has-errors .fjs-input-group {
+      outline: 2px solid var(--cds-text-error);
+      outline-offset: -2px;
+    }
+  }
+`;
+
+const DROPDOWN_STYLES = css`
+  ${({ theme }) =>
+    css`
+      .fjs-container {
+        .fjs-form-field-taglist .fjs-taglist-anchor .fjs-dropdownlist,
+        .fjs-form-field-datetime .fjs-timepicker-anchor .fjs-dropdownlist,
+        .fjs-form-field-select .fjs-select-anchor .fjs-dropdownlist {
+          margin: 0;
+          max-height: ${rem(264)};
+          border: none;
+          background-color: var(--cds-layer, var(--cds-layer-01));
+          overflow-y: auto;
+          cursor: pointer;
+          border-radius: 0;
+          box-shadow: 0 2px 6px var(--cds-shadow);
+
+          & .fjs-dropdownlist-item {
+            border: none;
+            box-sizing: border-box;
+            padding: 0;
+            margin: 0 ${theme.spacing05};
```

Comment:
Instead of using interpolation you can just use CSS vars. I left because I was laxy
```suggestion
            margin: 0 var(--cds-spacing-05);
```